configs/nextflow.ngp.config

```diff
@@ -9,9 +9,10 @@ params {
 	mlstBlastDb = "${root}/assets/mlst_db/blast"
 	pubMlstDb = "${root}/assets/mlst_db/pubmlst"
 	krakenDb = "/gms-home/isasyl/code/JASEN" //edit if useKraken = true
+	hostileIdx = "${root}/assets/hostile_db/human-t2t-hla"
```

Comment:
Same as in the base config. Shouldnt this be removed?

Reply:
At the moment, no. Everyone is still using their own config regardless of the base. The goal at some point is to have one main config and import region-specific settings as a profile. But for now, this will have to do.

Reply:
Indeed